src/cascadia/TerminalApp/AppCommandlineArgs.cpp

```diff
@@ -417,6 +417,11 @@ void AppCommandlineArgs::_addNewTerminalArgs(AppCommandlineArgs::NewTerminalSubc
                                                                   _startingTabColor,
                                                                   RS_A(L"CmdTabColorArgDesc"));
 
+    subcommand.suppressApplicationTitleOption = subcommand.subcommand->add_flag(

```

Comment:
Should we add some kind of shorthand version of this? `suppressApplicationTitle` is one of the longest named settings we have, I think haha.

Reply:
I am not sure, I usually write those once (in startupActions 😛 )

Reply:
Yea, I say let's leave it for now, and if someone _really_ wants a single-character flag for it in the future, and has a good idea, then we can address it then.